t/uni/variables.t

```diff
@@ -320,7 +321,7 @@ for my $i (0x100..0xffff) {
 EOP
     is($@, '', q{$$1 parses correctly});
 
-    for my $chr ( q{@}, "\N{U+FF10}", "\N{U+0300}" ) {
+    for my $chr ( "\N{U+FF10}", "\N{U+0300}" ) {
```

Comment:
I see you moved `@` to its own block

Reply:
Yes, because unlike in blead it is now generating a warning which needs to be either captured or suppressed.  We have to try to understand *why* a warning is now being generated is some places and not others.

Reply:
Yes, it now generates a warning which we have to either capture or suppress.